app/javascript/containers/NavigationBar/BusMap.jsx

```diff
@@ -62,4 +66,18 @@ BusMap.propTypes = {
   buildingId: PropTypes.number.isRequired
 };
 
-export default BusMap;
+function mapStateToProps(state, ownProps) {
+  return {
+    // number of undelegated questions, used to decide when to disable navigation in delegate mode
+    numUnanswered: numUnanswered(ownProps.buildingId, state)
+  }
+}
+
+function mapDispatchToProps(dispatch) {
+  return {};
+}
+
+export default connect(
+  mapStateToProps,
```

Comment:
The indentation here is a little weird - it's fine this time, but please install prettier.io and have it automatically format your code in the future!